### ISSUE TITLE
t&c and privacy policy pages are dead links

### ISSUE THREAD

**Issue by felixangell** [HandsFree/beaconing-teacher-ui#74]
## Issue summary
the links are dead

mantis issue:
281

**Comment by juddus** [comment]
Need to create a privacy policy and t&c beforehand

**Comment by felixangell** [comment]
Yeah I figured we don't really have one... this isn't our issue though I presume? Is this to do with our company or someone elses

**Comment by juddus** [comment]
Think someone else would make one, but we'd have to tell them the information we store etc. for the privacy policy
